specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2020-06-01-preview/nova.json

```diff
@@ -0,0 +1,1051 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Cosmos DB - Managed Cassandra",
+    "description": "Azure Cosmos DB Database Service Managed Cassandra REST API",
+    "version": "2020-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "Impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.DocumentDB/cassandraClusters": {
```

Comment:
AFAIK, cassandraCluster is a tracked resource. Not sure if we support subscription move for it. If not we would need to. RPC here[[https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc003-tracked-resource-types-must-support-move](https://armwiki.azurewebsites.net/api_contracts/guidelines/rpc.html#rpc003-tracked-resource-types-must-support-move)]

Another requirement for tracked resources is they must support Tags and those Tags must support Patch.

Reply:
We don't yet, but I'll make it pretend to, and we'll just throw errors until we have it implemented.